src/locales/en.json

```
@@ -181,8 +181,10 @@
   "FeaturedPage": {
     "headerLabel": "Today"
   },
-  "filter_menu_creator": "Filter",
+  "filter_menu_creator": "All Creators",
+  "filter_menu_creator_selected": "{num} Creator(s)",

```

Can use `$tc()`
Ref: https://kazupon.github.io/vue-i18n/guide/pluralization.html#pluralization
```suggestion
  "filter_menu_creator_selected": "{num} Creator | {num} Creators",
```